src/components/atoms/button/button.scss

```
@@ -0,0 +1,37 @@
+@use "../../variables" as * ;
+
+.btn-b1{
```

```suggestion
.button{
```
.btn だとbootstrapのスタイルがついてしまうということでこのクラス名にしてくれたのだと拝察します。
b1に深い意味があるわけでなければ、とりあえずbuttonというクラス名で大丈夫だと思います。